supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc

```diff
@@ -324,12 +324,22 @@ There is no functional difference between the first server that was installed an
 
 [[ingress]]
 ==== image:images/yes.png[yes] Ingress (noun)
-*Description*: In Red Hat OpenShift, _Ingress_  is an API object developers can use to expose services through an HTTP(S) aware load balancing and proxy layer through a public DNS entry. The Ingress resource might further specify TLS options and a certificate, or specify a public CNAME that the OpenShift Ingress Controller should also accept for HTTP and HTTPS traffic. An administrator typically configures their Ingress Controller to be visible outside the cluster firewall and might also add additional security, caching, or traffic controls on the service content.
+*Description*: In Red Hat OpenShift, _Ingress_  is an API object developers can use to expose services through an HTTP(S) aware load balancing and proxy layer through a public DNS entry. The `Ingress` resource might further specify TLS options and a certificate, or specify a public CNAME that the OpenShift Ingress Controller (`IngressController`) should also accept for HTTP and HTTPS traffic. An administrator typically configures their Ingress Controller to be visible outside the cluster firewall and might also add additional security, caching, or traffic controls on the service content.
 
 *Use it*: yes
 
 *Incorrect forms*:
+"Ingress" alone, when referencing the `Ingress` resource.
```

Comment:
This is confusing because the entry is for "Ingress". Move to the description.

Reply:
I changed this to Ingress without markup when referencing the `Ingress` resource or `IngressController`